cabby/data/metagraph/utils.py

```diff
@@ -55,6 +55,12 @@ def convert_pandas_df_to_metagraph(
       nx.set_node_attributes(g, attribute_dict, metadata_name)
   return g
 
+def get_osmid_from_wd_relations_row(row: pd.Series):
```

Comment:
Add return type and one-liner docstring.

Reply:
Fixed by 8cb04ea4046158db624b1350a62c500a84503ef6